kamailio/recordings/templates/emailBrands_body.tmpl

```diff
@@ -1,28 +0,0 @@
-This is an auto-generated message.
```

Comment:
I would keep templates. What do you think about it?

Reply:
I'm ok with that, I will move them to `microservices/recordings/templates/` until we implement #342 someday.

Thanks for the review :)